apps/etterlatte-tidshendelser/.nais/dev.yaml

```diff
@@ -21,7 +21,6 @@ spec:
   readiness:
     initialDelay: 5
     path: /isready
-  leaderElection: true
```

Comment:
Jeg har satt appen til 1 pod inntil videre, men i prinsippet ønsker jeg at _alle_ apper skal ha minimum 2 podder, både i prod og dev. Men jeg innser at jeg har ikke nok forståelse for R&R og om det setter noe begrensninger. Om vi klarer å slå sammen noen av appene tenker jeg i hvert fall det bør være slik.

Reply:
Som nevnt på pm er det bare et spørsmål av hvor mye dataen haster, det blir nedetid selvom man har 2 pods om man bruker leaderElection også. I tillegg til worst case problemer med parallelle spørringer som kan henge om man kaller behandlings metoder med intransaction.